modules/custom-email/custom-email.php

```diff
@@ -170,6 +178,33 @@ public function apply_retrieve_pass_filters() {
 	}
 
 	/**
+	 * Applies all change password mail filters
+	 *
+	 * Callback for "password_change_email" hook in Theme_My_Login::password_change_email()
+	 *
+	 * @see Theme_My_Login::password_change_email()
+	 * @since 6.0
+	 * @access public
+	 */
+	public function apply_pass_change_filter( $pass_change_email, $user, $userdata ) {
+		 $pass_change_email['headers']= array(
+			 $this->get_option( array( 'pass_change', 'mail_from'         ) ),
+			 $this->get_option( array( 'pass_change', 'mail_from_name'    ) ),
+			 $this->get_option( array( 'pass_change', 'mail_content_type' ) )

```

Comment:
I finally found why this didn't work for me for HTML mails. Since 'mail_content_type_filter' is not used here, 'text/' is not prepended. Therefore, the header isn't correct and the HTML mails aren't processd as HTML.